src/Chapter16.Tests/Listing16.15.Tests.cs

```diff
@@ -1,22 +1,98 @@
+                    SkipLast, SkipWhile, Sum, Take, TakeLast, TakeWhile, ThenBy, 
+                    ThenByDescending, ToArray, ToDictionary, ToHashSet, ToList, 
+                    ToLookup, TryGetNonEnumeratedCount, Union, UnionBy, Where, Zip, 
+                    """,
+                // Version 7.0
+                _ => """
+                    Aggregate, All, Any, Append, AsEnumerable, Average, Cast, 
+                    Chunk, Concat, Contains, Count, DefaultIfEmpty, Distinct, DistinctBy, 
+                    ElementAt, ElementAtOrDefault, Empty, Except, ExceptBy, First, 
+                    FirstOrDefault, GroupBy, GroupJoin, Intersect, IntersectBy, Join, 
+                    Last, LastOrDefault, LongCount, Max, MaxBy, Min, MinBy, OfType, 
+                    Order, OrderBy, OrderByDescending, OrderDescending, Prepend, Range, 
+                    Repeat, Reverse, Select, SelectMany, SequenceEqual, Single, 
+                    SingleOrDefault, Skip, SkipLast, SkipWhile, Sum, Take, TakeLast, 
+                    TakeWhile, ThenBy, ThenByDescending, ToArray, ToDictionary, ToHashSet, 
+                    ToList, ToLookup, TryGetNonEnumeratedCount, Union, UnionBy, Where, Zip, 
+                    """
+            };
+
+
             // New lines added for output formatting in the chapter.
-            string expected =
-@"Aggregate, All, Any, Append, AsEnumerable, Average, Cast, Concat, Contains, 
-Count, DefaultIfEmpty, Distinct, ElementAt, ElementAtOrDefault, Empty, 
-Except, First, FirstOrDefault, GroupBy, GroupJoin, Intersect, Join, Last, 
-LastOrDefault, LongCount, Max, Min, OfType, OrderBy, OrderByDescending, 
-Prepend, Range, Repeat, Reverse, Select, SelectMany, SequenceEqual, 
-Single, SingleOrDefault, Skip, SkipLast, SkipWhile, Sum, Take, TakeLast, TakeWhile, ThenBy, 
-ThenByDescending, ToArray, ToDictionary, ToHashSet, ToList, ToLookup, Union, Where, Zip, ";
+            string expected = """
+                    Aggregate, All, Any, Append, AsEnumerable, Average, Cast, 
+                    Chunk, Concat, Contains, Count, DefaultIfEmpty, Distinct, DistinctBy, 
+                    ElementAt, ElementAtOrDefault, Empty, Except, ExceptBy, First, 
+                    FirstOrDefault, GroupBy, GroupJoin, Intersect, IntersectBy, Join, 
+                    Last, LastOrDefault, LongCount, Max, MaxBy, Min, MinBy, OfType, 
+                    Order, OrderBy, OrderByDescending, OrderDescending, Prepend, Range, 
+                    Repeat, Reverse, Select, SelectMany, SequenceEqual, Single, 
+                    SingleOrDefault, Skip, SkipLast, SkipWhile, Sum, Take, TakeLast, 
+                    TakeWhile, ThenBy, ThenByDescending, ToArray, ToDictionary, ToHashSet, 
+                    ToList, ToLookup, TryGetNonEnumeratedCount, Union, UnionBy, Where, Zip, 
+                    """;
+
+            TestContext.WriteLine(Shared.NetCore.GetNetCoreVersion());
```

Comment:
This code appears to invoke `Shared.NetCore.GetNetCoreVersion()` several times. Can we cache the return in a local.
```suggestion
            string dotnetVersion = Shared.NetCore.GetNetCoreVersion(); 
            TestContext.WriteLine(dotnetVersion);
```